src/tests/plugins/test_sendmail.py

```diff
@@ -0,0 +1,176 @@
+import datetime
+
+import pytest
+
+from django.core import mail as djmail
+from django.utils.timezone import now
+
+from pretix.base.models import (
+    Event, EventPermission, Item, ItemCategory, Order, OrderPosition,
+    Organizer, OrganizerPermission, User,
+)
+
+
+@pytest.fixture
+def event():
+    '''Returns an event instance'''
```

Comment:
Just a very minor style thing here, please use ``"""`` instead of ``'''`` for docstrings.